wpeverest/everest-forms
# Tweak - Test email input and button design

### All Submissions:



### Changes proposed in this Pull Request:

Previously, the send test email button was under the title of **send test email** and another input field was there to input the recipient email address which made the design of **send test email** more ambiguous. This PR fixes the design of **test email in both reporting and email section.** 

Closes # .

### How to test the changes in this Pull Request:

1. Go to **Global Settings > Email > Send Test email**
2. Check whether the email is being sent accordingly to the specified email address.
3. Go to **Global Settings >  Reporting** and sent the test email
4. And check whether the email is being sent accordingly or not.

### Types of changes:



### Other information:



### Changelog entry
> Tweak - Test email input and button design
